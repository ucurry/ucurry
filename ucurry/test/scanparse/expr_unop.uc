-- def -> expr -> unop
~1;
not true;
not false;
hd [1, 2, 3];
tl [2, 3, 4];
