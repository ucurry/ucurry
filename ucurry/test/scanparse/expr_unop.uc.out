Printing Roundtrip
-1;
!true;
!false;
hd[1, 2, 3];
tl[2, 3, 4];
